terraform/tfschema/types_terraform.go

```diff
@@ -52,10 +56,10 @@ var (
 	SchemaProvisionTokenV2 = GenSchemaProvisionTokenV2()
 	// SchemaMetaProvisionTokenV2 is schema metadata for ProvisionTokenV2 specifies provisioning token
 	SchemaMetaProvisionTokenV2 = GenSchemaMetaProvisionTokenV2()
-	// SchemaRoleV3 is schema for RoleV3 represents role resource specification
-	SchemaRoleV3 = GenSchemaRoleV3()
```

Comment:
Is this a backwards compatible change? This looks like we're removing V3 role from terraform spec completely unless I'm missing something. What happens if somebody is still using V3 roles?

Reply:
@gzigzigzeo see the question above? both on api and V3 change

Reply:
@klizhentas @r0mant They will be automatically upgraded to v4. I've tested this. There's no way I can use both because API has no RoleV3 struct anymore, otherwise I would keep both.